src/views/virtualmachines/list/components/VirtualMachineRow/VirtualMachineRowLayout.tsx

```diff
@@ -42,7 +42,7 @@ const VirtualMachineRowLayout: React.FC<
         activeColumnIDs={activeColumnIDs}
         className="dropdown-kebab-pf pf-c-table__action"
       >
-        <VirtualMachineActions />
+        <VirtualMachineActions vm={obj} />
```

Comment:
it will be more readable and less error prone to use `LazyActionMenu` directly, `VirtualMachineActions` is unnecessary complexity.

Reply:
will be fixed in a follow-up PR